dask/array/random.py

```diff
@@ -71,7 +71,7 @@ def _wrap(self, func, *args, **kwargs):
             shapes += [size]
         # broadcast to the final size(shape)
         size = broadcast_shapes(*shapes)
-        chunks = normalize_chunks(chunks, size)
+        chunks = normalize_chunks(chunks, size, dtype=np.float64)  # ideally we would use dtype here
```

Comment:
Re the comment: why not use the dtype, if provided? Persumabley `dtype=kwargs.get('dtype', np.float64)`

Reply:
Fixed